src/module/actor/sheet/base.ts

```diff
@@ -719,7 +726,7 @@ export abstract class ActorSheetPF2e<TActor extends ActorPF2e> extends ActorShee
                     actor,
                     {},
                     async (heightenedLevel, itemType, spell) => {
-                        if (!(itemType === "scroll" || itemType === "wand")) return;
+                        if (!isSpellConsumableItemType(itemType)) return;
```

Comment:
You should probably remove this check and make the function's type into SpellConsumableItemType.